static-site/src/components/splash/index.jsx

```diff
@@ -211,61 +213,179 @@ export default Splash;
 /*** SHOWCASE ***/
 
 const UrlShowcaseTile = ({ card }) => {
+
+  /* Narrative detection works for all three sources:
+  1. Core: /narratives/<narrative path>
+  2. Community: /community/narratives/<owner>/<narrative path>
+  3. Groups: /groups/<group>/narratives/<narrative path>
+
+  Including slashes in the check prevents false positives.
+  */
+  const isNarrative = card.url.includes('/narratives/');
+
   return (
     <CardOuter>
       <CardInner>
         <a href={card.url}>
-          <CardTitle $squashed>
+          <CardTitle>
             {card.name}
           </CardTitle>
-          <CardImgWrapper filename={card.img}/>
+          <CardImgContainer>
+            <CardImgWrapper filename={card.img}/>
+            <InfoIcons>
+              <CardSourceIcon url={card.url} isNarrative />
+              {isNarrative && <NarrativeIcon />}
+            </InfoIcons>
+          </CardImgContainer>
         </a>
+        <CardDescription>
+          {card.description}
+        </CardDescription>
       </CardInner>
     </CardOuter>
   )
 }
 
-const cardWidthHeight = 160; // pixels
+const cardWidth = 220; // pixels
+const cardHeight = 285; // pixels
```

Comment:
This height works when the description fits on two lines, but (e.g.) I wanted to expand the description for the cattle-flu card but the text becomes clipped (1st screenshot)

A long title which ends up as >1 line makes things look even worse (2nd screenshot)

None of the current entries have this problem, so we can defer work on this to a separate issue.

<img width="186" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/10a6823d-02b0-4c03-a996-136705597848">

<img width="183" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/408fb252-68e8-4c0e-946a-2144edc1b61e">

Reply:
In writing these titles I made sure to stick to one line. Otherwise we have an alignment issue across cards. I think it's fair to demand these be short enough to fit on one line.